django_tasks/backends/base.py

```diff
@@ -92,3 +93,9 @@ def close(self) -> None:
         """
         # HACK: `close` isn't abstract, but should do nothing by default
         return None
+
+    def check(self, **kwargs: Any) -> List[CheckMessage]:
+        raise NotImplementedError(
```

Comment:
Issue: This means the checks will fail by default when tasks is used. Instead, this should be a noop by default, or we should explicitly add a noop to each backend we provide.